easybuild/easyconfigs/p/pgi/pgi-17.10-gcc-5.4.0-2.26.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'Bundle'
+
+name = 'PGI'
```

Comment:
The eb file name here does not match the case of the package name , so you might have problems when using this eb as dependency (one more reason for having this file in the same pr as netcdf).